### PR TITLE
Minor farm list updates

### PR DESCRIPTION
**Pull request by thomasgwatson**
https://github.com/Hylozoic/hylo-evo/issues/1462

### CODE REVIEW

File: lib/constants.js [hunk 1]
```diff
@@ -446,6 +446,14 @@ exports.PREFERRED_CONTACT_METHODS = [
...
+  {
+    value: 'grains',
+    label: 'Grains'
+  },
+  {
+    value: 'other_row_crops',
+    label: 'Other row crops'
+  },
   {
     value: 'grains_other',
```
Comment (tibetsprague) [inline lib/constants.js:458]: so this doesnt need to appear in the schema?

File: lib/constants.js [hunk 2]
```diff
@@ -446,6 +446,14 @@ exports.PREFERRED_CONTACT_METHODS = [
 ]
 
 exports.PRODUCT_CATAGORIES = [
+  {
+    value: 'grains',
+    label: 'Grains'
+  },
+  {
+    value: 'other_row_crops',
```
Comment (tibetsprague) [inline lib/constants.js:454]: this doesnt need to appear in the schema?

File: lib/constants.js (hunk 1 above)
Comment (thomasgwatson) [reply lib/constants.js:458]: Notionally no, but leaving it on makes this backwards compatible with existing farm groups.

File: lib/constants.js (hunk 2 above)
Comment (thomasgwatson) [reply lib/constants.js:454]: Notionally no, but leaving it on makes this backwards compatible with existing farm groups.

**Review by tibetsprague — APPROVED** [review]

**Pull request merged**